Modification in some functionality and solved some minor bugs.

Click2cloud-Eros opened the pull request:
**What type of PR is this?**
1) bug fix
2) enhancement

**What this PR does / why we need it?**
This PR included some modification and bugs fixes.

**Which modification and issue(s) this PR fixes:** 
1) Merge some array attributes (i.e. titles, drawables, values, percents, colors) into single ArrayList attribute (i.e. NodeList).
2) Add **enabledMultiColorRegion** and **enabledCircularOutline** attribute for xml.
3) Solved last button cut issue.
4) Solved setCount issue.
5) Change functionality of **Load Animation** button.
6) Add and remove Junit testcases as per new implementation.
7) Update readme file content as per new implementation.
8) Move Attributes to new file.

Click2cloud-Eros committed:
Move attributes into new class.
Solved final button cut issue.
Solved load animation bug.
Add enabledCircularOutline attribute and update same in Readme.md file.

Click2cloud-Eros committed:
Modification in some functionality.
Add and remove Junit testcases as per new implementation.
Update Readme file content as per new implementation.

Click2cloud-Eros committed:
Update Readme file.

Click2cloud-Eros committed:
update screenshot in readme file.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/applibgroup/XRadarView?pullRequest=5) <br/>All committers have signed the CLA.

kanaksony commented on xradar/src/main/java/com/orzangleli/radar/XRadarView.java:
```diff
@@ -24,53 +21,52 @@
     private float radarPercent = 0.7f;
 
+    // When the gradient is turned on, the color at the center of the circle
     private Color startColor = new Color(RgbPalette.parse("#80FF0000"));
```
Please move this and similar in resources

kanaksony reviewed (changes requested):
Please add Copyright for new code files.
Add colors code in resources instead of code.
Utils.java# getPathById() shall be optimized like - 
` 
public static String getPathById(Context context, int id) {
        if (context != null) {
            ResourceManager manager = context.getResourceManager();
            if (manager != null) {
                try {
                    return manager.getMediaPath(id);
                } catch (IOException | NotExistException | WrongTypeException e) {
                    //TO-DO: Handle exception here
                    return "";
                }
            }
        }
        return "";
    }
`

Node - There are some methods never called

Click2cloud-Eros committed:
Merge branch 'applibgroup:master' into main

Click2cloud-Eros commented:
> Please add Copyright for new code files. Add colors code in resources instead of code. Utils.java# getPathById() shall be optimized like - ` public static String getPathById(Context context, int id) { if (context != null) { ResourceManager manager = context.getResourceManager(); if (manager != null) { try { return manager.getMediaPath(id); } catch (IOException | NotExistException | WrongTypeException e) { //TO-DO: Handle exception here return ""; } } } return ""; }`
> 
> Node - There are some methods never called

I seen, I have already added copyright in new files.
And other changes I will do.

Click2cloud-Eros committed:
Move colors to resources.
optimized getPathById method.
Handle null pointer exception in getPixleMap Method.

kanaksony reviewed (approved):
Looks good

sharathks84 reviewed (approved):
Okay Approved
